modules/service-fips.sh

```diff
@@ -50,6 +54,71 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_updates_enable() {
+    local token="$1"
+    local bypass_prompt="$2"
+
+    local fips_configured=0
+    local fips_kernel_version=0
+    local result=0
+
+    _fips_updates_is_enabled || result=$?
+    if [ $result -eq 0 ]; then
+        error_msg "FIPS-UPDATES repository is already enabled."
+        return;
+    fi
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
+
+    echo "Installing updates from FIPS-UPDATES repository will take the system out of FIPS compliance."
+    if [ "$bypass_prompt" -ne 1 ]; then
+        if ! prompt_user 'Do you want to proceed?'; then
+            error_msg "Aborting updating FIPS packages..."
+            error_exit service_already_enabled
```

Comment:
I think you meant for this error_exit to be elsewhere,  not when bypassing the prompt

Reply:
oops. fixing it

Reply:
Done